### PR TITLE
MGMT-7847: Add a deprecation warning for the VerifyCidr option

### PR DESCRIPTION
**Pull request by flaper87**
Signed-off-by: Flavio Percoco <flavio@redhat.com>

### CODE REVIEW

File: src/apivip_check/apivip_check.go [hunk 1]
```diff
@@ -39,6 +39,7 @@ func CheckAPIConnectivity(checkAPIRequestStr string, log logrus.FieldLogger) (st
 	}
 
 	if checkAPIRequest.VerifyCidr {
+		log.Warnf("The VerifyCidr ()`verify_cidr`) is being deprecated. CIDR verification is done by Assisted Service directly")
```
Comment (omertuc) [inline src/apivip_check/apivip_check.go:42]: uneven ()

Comment (flaper87) [reply src/apivip_check/apivip_check.go:42]: :facepalm:

File: src/apivip_check/apivip_check.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ func CheckAPIConnectivity(checkAPIRequestStr string, log logrus.FieldLogger) (st
 	}
 
 	if checkAPIRequest.VerifyCidr {
+		log.Warnf("The VerifyCidr (`verify_cidr`) is being deprecated. CIDR verification is done by Assisted Service directly")
```
Comment (mkowalski) [inline src/apivip_check/apivip_check.go:42]: I'm going to LGTM anyway, but not sure if backtick (`\``) printed to the terminal and/or log file is not crashing something

Comment (flaper87) [reply src/apivip_check/apivip_check.go:42]: Let me remove it, we don't want things to crash

**Pull request merged**